code/datums/holocall.dm

```diff
@@ -0,0 +1,154 @@
+#define HOLOPAD_MAX_DIAL_TIME 200

```

Comment:
Might be best for it to be a `var/global` since I doubt there'd be many reasons to ever edit a single instance